core/tests/test_utils.py

```diff
@@ -161,6 +164,127 @@ def check_image_png_or_webp(image_string):
     return False
 
 
+class EmailMessageMock(object):
+    """Mock for google.appengine.api.email message."""
```

Comment:
This seems odd. We're getting rid of google.appengine.api.email altogether, so why are we still mocking it?

Instead you should mock mailgun (based on the current API for that) since you want to have a "local stub" for that. Any trace of anything related to google.appengine.api.email should disappear henceforth.

Reply:
Done